apps/frontend/src/views/Compare.vue

```diff
@@ -575,6 +575,43 @@ export default class Compare extends Props {
     }
     this.changeChartState();
   }
+
+  /**
+   * The title to override with
+   */
+  get curr_title(): string | undefined {
+    if (this.file_filter.length == 1) {
+      let file = InspecDataModule.allEvaluationFiles.find(
+        f => f.unique_id === this.file_filter[0]
+      );
+      if (file) {
+        return 'Comparison View (' + file.filename + ' selected)';
+      }
+    }
+    if (this.file_filter.length > 1) {
+      return (
+        'Compare View (' + this.file_filter.length + ' results selected' + ')'
+      );
+    } else {
+      return 'Compare View (No files selected)';
+    }
+  }
```

Comment:
This may be slightly simpler

```javascript
  get curr_title(): string {
    let returnText = 'Comparison View'
    if (this.file_filter.length == 1) {
      let file = InspecDataModule.allEvaluationFiles.find(
        f => f.unique_id === this.file_filter[0]
      );
      if (file) {
        returnText += ` (${file.filename} selected)`;
      }
    }
    else {
      returnText += ` (${this.file_filter.length} results selected)`;
    }
    return returnText;
  }
```

Reply:
I'll use this algorithm for the Profiles, Results, and Compare vue files.